pol-core/pol/network/pktboth.h

```diff
@@ -678,7 +681,7 @@ struct PKTBI_BF_14
   u16 unk;  // always 00 01
   u32 serial;
   u8 numentries;
-  PKTBI_BF_14_ENTRIES entries[1];
+  PKTBI_BF_14_ENTRIES entries[( MAXBUFFER - 7 ) / 8];
```

Comment:
this works, but would max(u8)*8 make more sense?

Reply:
I guess it never changes, the correct thing would be maybe to have a size of(entries) instead of 8 :)